util/linux_install.sh

```diff
@@ -4,6 +4,20 @@
 
 GENTOO_WARNING="This script will make a USE change in order to ensure that that QMK works on your system. All changes will be sent to the the file /etc/portage/package.use/qmk_firmware -- please review it, and read Portage's output carefully before installing any packages on your system. You will also need to ensure that your kernel is compiled with support for the keyboard chip that you are using (e.g. enable Arduino for the Pro Micro). Further information can be found on the Gentoo wiki."
 
+read -rd '' SLACKWARE_WARNING << EOF
```

Comment:
```suggestion
```
Remove the heredoc starting with this line and ending with the `EOF` line below.